docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md

```diff
@@ -1,196 +1,229 @@
+8. Review the profile layers and customize parameters as desired in the YAML files that display when you select a layer. You can configure custom OpenID Connect (OIDC) for Azure clusters at the Kubernetes layer. Check out [Configure OIDC Identity Provider](../../../integrations/kubernetes.md#configure-oidc-identity-provider) for more information.
 
-:::
-
-<br />
+  :::caution
 
-|**Parameter**| **Description**|
-|-------------|---------------|
-| **Subscription** | From the drop-down menu, select the subscription that will be used to access Azure Services.|
-| **Region** | Select a region in Azure in which the cluster should be deployed.|
-| **Resource Group** | Select the Azure resource group in which the cluster should be deployed.|
-| **Storage Account** | Optionally provide the storage account. Review the [Azure Storage section](architecture#azure-storage) for a custom storage use cases. |
-| **Storage Container**| Optionally provide the Azure storage container. Review the [Azure Storage section](architecture#azure-storage) for a custom storage use cases.|
-| **SSH Key** | The public SSH key for connecting to the nodes. Review Microsoft's [supported SSH](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/mac-create-ssh-keys#supported-ssh-key-formats) formats. |
-| **Static Placement** | By default, Palette uses dynamic placement, in which a new VPC with a public and private subnet is created to place cluster resources for every cluster. These resources are fully managed by Palette and deleted when the corresponding cluster is deleted. <br /> If you want to place resources into pre-existing VPCs and subnets, you can enable the **Static Placement** option. Review the [Static Placement](#static-placement-table) table below for available parameters for static placement.|
-|**Update worker pools in parallel**| Check the box to concurrently update the worker pools.|
-|**Private API Server LB**|This option applies when the cluster is deployed via the [Azure Private Endpoint](gateways.md). You can enable this option if your API Server must have private access. Review the [Private API Server LB](#private-api-server-lb-table) table below for more details.|
-|**Update worker pools in parallel**|If you have multiple worker pools, select the check box to enable simultaneous upgrade of all the pools. The default is sequential upgrade.|
+  All the OIDC options require you to map a set of users or groups to a Kubernetes RBAC role. To learn how to map a Kubernetes role to users and groups, refer to [Create Role Bindings](../../cluster-management/cluster-rbac.md/#create-role-bindings).
 
-#### Static Placement Table
+  :::
 
-| **Parameter**              | **Description** |

```

Comment:
I'm not sure I agree with the conversion of this into a bullet. It looks out of place as a stand-alone bullet.

![CleanShot 2023-11-21 at 19 15 22@2x](https://github.com/spectrocloud/librarium/assets/29551334/92092bf4-f7f7-43ab-8757-2ffa38f297c8)